CHANGELOG.md

```diff
@@ -11,6 +11,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
  - Support to search multiple markets at once.
  - Support to search all available Spotify markets.
+ - Support to test whether the current user is following certain
+ users or artists
+ - Proper replacements for all deprecated playlist endpoints
+ (See https://developer.spotify.com/community/news/2018/06/12/changes-to-playlist-uris/ and below)
+
+### Deprecated
+
+- `user_playlist_change_details` in favor of `playlist_change_details`
+- `user_playlist_unfollow` in favor of `current_user_unfollow_playlist`
+- `user_playlist_add_tracks` in favor of `playlist_add_tracks`
```

Comment:
Will the endpoints work for episodes as they are? I am thinking that there may be a problem at client.py:1023 if a URI or URL is given because it is looking for tracks.

```py
... self._get_uri("track", tid) ...
```

Reply:
Yes it looks like it will be fine. Strangely the endpoint titles use "items" but the endpoints still have "tracks" in them even though it refers to both tracks and episodes

    GET https://api.spotify.com/v1/playlists/{playlist_id}/tracks

https://developer.spotify.com/documentation/web-api/reference/playlists/get-playlists-tracks/